packages/react-utils/src/components/GenericDetailsView/ResourceDetails/tests/index.test.tsx

```diff
@@ -18,9 +18,13 @@ const props = {
     Alias: 'Clinic',
     'Administrative Level': 2,
   },
+  bodyDataFunction: () => <div>Additional Content</div>, // Added bodyDataFunction
```

Comment:
The prop is not `bodyDataFunction`should be `bodyData`. In this test case we are already testing what happens when that is an object, I think you ought to add another test case that now passes the value of bodyData as a render prop.